src/styles/components/_home-search-box.scss

```
@@ -1,18 +1,65 @@
 @import '../abstracts/_abstracts-dir';
 
 .home-search-box {
-  @include flex-center();
+  display: flex;
+  width: 80%;
 
-  &__button {
-    @include button-darken-effect($picton-blue);
+  background-color: $light;
+  box-shadow: rem(5px 10px 20px) 0px box-shadow-opacity(0.16);
+
+  @include respond-above('md') {
+    width: 90%;
+  }
 
-    text-decoration: none;
+  &__input {
+    width: 100%;
+    box-sizing: border-box;
     padding: rem(5px 10px);
-    border-radius: rem(3px);
-    background-color: $picton-blue;
+
+    color: $dove-gray;
+
+    border: none;
+    font-size: rem(14px);
+    line-height: 1.23;
+
+    @include respond-above('sm') {
+      padding: rem(10px 20px);
+      font-size: rem(16px);
+    }
+
+    @include respond-above('md') {
+      padding: rem(20px 40px);
+      font-size: rem(20px);
+    }
+
+    @include respond-above('lg') {
+      padding: rem(35px 60px);
+      font-size: rem(24px);
+    }
   }
 
-  &__text {
-    color: $light;
+  &__button {
+    width: rem(28px);
+    padding: 0;
+    background-color: $light;
+    border: none;
+    background: {
+      image: url('/icons/search-icon.svg');
+      repeat: no-repeat;
+      size: 70%;
+      position: left;
+    }
```

We can have color inside `background` object property.
```suggestion
    border: none;
    
    background: {
      color: $light;
      image: url('/icons/search-icon.svg');
      repeat: no-repeat;
      size: 70%;
      position: left;
    }
```